Swat/SwatTextareaEditor.php

```diff
@@ -335,7 +340,7 @@ protected function getInlineJavaScript()
         $lines = array();
         foreach ($this->getConfig() as $name => $value) {
             if (is_string($value)) {
-                $value = SwatString::quoteJavaScriptString($value);
+                $value = "'{$value}'";
```

Comment:
why did you remove `SwatString::quoteJavaScriptString`? That will escape any characters and avoid the possibility of JS injection.

Reply:
I think I removed it when I was trying something out that required some inline javascript and this was turning it into a string, I've changed it back